db/bigtable.go

```diff
@@ -1016,9 +1034,19 @@ func (bigtable *Bigtable) GetValidatorMissedAttestationsCount(validators []uint6
 	for validator, attestations := range data {
 		missed := len(attestations)
 		if missed > 0 {
```

Comment:
```suggestion
		if len(attestations) == 0 {
			continue
		}
```
**Minor:** This would be nicer.

Reply:
yep thats nicer